src/main/java/uk/org/tombolo/importer/barkingindices/ONSAverageAttainmentImporter.java

```diff
@@ -0,0 +1,153 @@
+package uk.org.tombolo.importer.barkingindices;
+
+import org.apache.poi.ss.usermodel.DataFormatter;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.org.tombolo.core.*;
+import uk.org.tombolo.core.utils.SubjectTypeUtils;
+import uk.org.tombolo.core.utils.SubjectUtils;
+import uk.org.tombolo.core.utils.TimedValueUtils;
+import uk.org.tombolo.importer.AbstractImporter;
+import uk.org.tombolo.importer.Config;
+import uk.org.tombolo.importer.ons.AbstractONSImporter;
+import uk.org.tombolo.importer.ons.OaImporter;
+
+import java.io.File;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Importer for Average Attainment score 8 per pupil by local authority created as part of the Barking and Dahenham manifesto.
+ * The data are relevant for state-funded schools only.
+ * The data are fetched from ONS Department for Education.
+ * Geography: Local authorities
+ * Unit of measurement: Please consult https://www.gov.uk/education/school-performance-measures
+ */
+public class ONSAverageAttainmentImporter extends AbstractImporter {
+    private static final String DATASOURCE = "https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/652293/SFR57_2017_LA__tables.xlsx";
+    private static final Logger log = LoggerFactory.getLogger(ONSAverageAttainmentImporter.class);
+
+    public ONSAverageAttainmentImporter(Config config) {
+        super(config);
+        try {
+            // Specifying the datasourceId. This will be used by the DC recipe
+            datasourceIds = Arrays.asList(getDatasourceSpec("ONSAverageAttainment").getId());
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+    // Instantiating the data Provider
+    protected static final Provider PROVIDER = new Provider(
```

Comment:
When you'll include this importer in the ons folder this will be inherited from the parent.